src/test/java/org/jfree/chart/entity/CategoryLabelEntityTest.java

```diff
@@ -33,39 +33,56 @@
  * Contributor(s):   -;
  *
  */
-
 package org.jfree.chart.entity;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import java.awt.geom.Rectangle2D;
+import nl.jqno.equalsverifier.EqualsVerifier;
+import nl.jqno.equalsverifier.Warning;
 
 import org.jfree.chart.TestUtils;
 import org.junit.jupiter.api.Test;
 
 /**
  * Tests for the {@link CategoryLabelEntity} class.
  */
-public class CategoryLabelEntityTest {
+public class CategoryLabelEntityTest
+{
```

Comment:
Nit: I'm not a big fan of curly braces on the next line, just because it wastes vertical screen space

Reply:
Ok, no problem! I thought I got all those taken care of but I missed a few.